### PR TITLE
remote actions refactor

### PR DESCRIPTION
**Pull request by benoitvidis**
Following https://github.com/kuzzleio/kuzzle/pull/359 and https://github.com/kuzzleio/kuzzle/pull/354

### CODE REVIEW

File: source/includes/plugins/_01-managing-plugins-cli.md [hunk 1]
```diff
@@ -91,84 +115,102 @@ There are multiple ways of changing a plugin configuration.
...
+ { npmVersion: '2.0.2',
+   activated: true,
+   config: 
+    { services: 
+       { file: 
+          { outputs: 
+             { error: { level: 'error', filename: 'kuzzle-error.log' },
+               warning: { level: 'warn', filename: 'kuzzle-warning.log' } },
+            addDate: true },
+         stdout: { level: 'info', addDate: true } },
```
Comment (stafyniaksacha) [inline source/includes/plugins/_01-managing-plugins-cli.md]: Why there is a stdout entry at this level ?

**Comment by ballinette** [comment]
+1 (waiting for related PRs to be merged before merging this one)

File: source/includes/plugins/_01-managing-plugins-cli.md (hunk 1 above)
Comment (benoitvidis) [inline source/includes/plugins/_01-managing-plugins-cli.md]: The indentation is a bit weird due to the fact that it is now directly taken from console.dir but the `stdout` entry is actually on the same level as `file`, which is the expected value.

I will reindent the output to be a little more readable.

File: source/includes/user_guide/_03-cli.md [hunk 2]
```diff
@@ -1,113 +1,162 @@
...
+      -s, --server               start Kuzzle in server mode (default)
+      -w, --worker               start Kuzzle in worker mode
+          --likeAvirgin          delete Kuzzle configuration and users from database
+          --fixtures <fixtures>  import some fixtures from file
+          --mappings <mappings>  load and apply mappings from file
 ```
+
+The `start` command starts a Kuzzle instance in the foreground.
+
+#### Server vs Worker
```
Comment (scottinet) [inline source/includes/user_guide/_03-cli.md]: Since we decided to wait for this change before releasing RC5, maybe this part should go.

**Pull request merged**